sygnal/gcmpushkin.py

```diff
@@ -348,7 +355,11 @@ async def _dispatch_notification_unlimited(self, n, device, context):
             # TODO: Implement collapse_key to queue only one message per room.
             failed = []
 
-            body = {"data": data, "priority": "normal" if n.prio == "low" else "high"}
+            # use the base body as a foundation for the body; this lets the
+            # Sygnal admin choose custom FCM options (e.g. content_available).
```

Comment:
I think this comment might be better suited to where we pull the options from the config. Perhaps with some rewording of `Use the fcm_options config dictionary as a foundation for the body...`